apps/app.near/widget/Components/Profile/Tab/NFTs.jsx

```diff
@@ -0,0 +1,126 @@
+const accountId = props.accountId || context.accountId;
+const thumbnails = props.thumbnails;
+
+if (!accountId) {
+  return <></>;
+}
+
+const f = fetch(`https://api.fastnear.com/v0/account/${accountId}/nft`);
+
+if (!f.ok) {
+  return "Loading";
+}
+
+const allNfts = f.body.contract_ids;
+
+const Wrapper = styled.div`
+  --template-column-gutters: 16px;
+  --template-columns: 1;
+  --template-column-compact-multiplier: 1;
+  --template-reduced-columns: 0;
+  --template-reduced-columns-multiplier: 1;
+
+  @media (min-width: 540px) {
+    --template-columns: 2;
+  }
+
+  @media (min-width: 992px) {
+    --template-columns: 3;
+  }
+
+  @media (min-width: 1140px) {
+    --template-columns: 4;
+  }
+
+  margin: var(--template-column-gutters) -12px 16px;
+
+  display: grid;
+  gap: var(--template-column-gutters);
+  grid-auto-rows: minmax(0px, 1fr);
+  grid-template-columns: repeat(
+    calc(
+      var(--template-columns) -
+        (
+          var(--template-reduced-columns) *
+            var(--template-reduced-columns-multiplier)
+        )
+    ),
+    minmax(0, 1fr)
+  );
+
+  .nft-card {
+    width: 100%;
+    height: 100%;
+    background-color: var(--bg1);
+    border: 1px solid var(--stroke);
+    border-radius: 1rem;
+    box-shadow: 0px 4px 8px -2px var(--shadow);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    overflow: hidden;
+    transition: all 0.3s ease 0s;
+
+    .nft-image-wrapper {
+      width: 100%;
+      padding-bottom: 100%;
+      position: relative;
+      background-color: #fff;
+      overflow: hidden;
+    }
+
+    .nft-image {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      object-fit: contain;
+    }
+
+    .nft-text {
+      width: 100%;
+      height: 100%;
+      padding: 8px;
+      color: var(--color);
+      max-height: 7em;
+      overflow: hidden;
+      font-family: "InterVariable", sans-serif;
+    }
+
+    .nft-title {
+      font-size: 16px;
+      color: var(--color);
+      margin-bottom: 4px;
+      font-family: var(--accent-font-family), InterVariable, sans-serif;
+      font-weight: bold;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+
+    .nft-description {
+      font-size: 16px;
+      font-family: "InterVariable", sans-serif;
+      height: 4.5em;
+      color: var(--color-muted);
+      white-space: break-word;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+  }
+`;
+
+return (
+  <Wrapper className="container-xl">
+    {allNfts.map((contractId, i) => (
```

Comment:
Can you replace this (and other instances that load and display data) with the Feed component?

https://near.social/devs.near/widget/Feed.demo

It can take "items" which is an array. The item will be passed to Item as props.

You can provide a Layout too, which is just a styled component

Reply:
I tried to use the Feed, but it gives error that props.index is not defined when you pass in the "items" prop